ui/admin/setup-edit.php

```diff
@@ -875,6 +878,7 @@
                 </div>
             </div>
             <!-- /#submitdiv -->
+            <?php do_meta_boxes( 'pods_edit', 'side', $pod ); ?>
```

Comment:
Tested and works, see https://github.com/pods-framework/pods/pull/3609/commits/5c2be5fb768a52e426822fcf3c0b1993f1b460a9